charmcraft/commands/build.py

```diff
@@ -254,6 +254,19 @@ def pack_charm_in_instance(
         charm_name = format_charm_file_name(
             self.metadata.name, self.config.bases[bases_index]
         )
+
+        # If building in project directory, use the project path as the working
+        # directory. The output charms will be placed in the correct directory
+        # without needing retrieval. If outputing to a directory other than the
+        # charm project directory, we need to output the charm outside the
+        # project directory and can retrieve it when complete.
+        if os.getcwd() == str(self.charmdir):
```

Comment:
```suggestion
        cwd = pathlib.Path(os.getcwd())
        if cwd == self.charmdir.absolute():
```

Reply:
Or `pathlib.Path.cwd()`:
```
>>> import pathlib
>>> pathlib.Path.cwd()
PosixPath('/home/claudio')
```

Reply:
I looked for this and could not find it, thanks for the pointer :-)

Reply:
Done! Thanks!